lib/dictionary_reader.rb

```diff
@@ -0,0 +1,16 @@
+class DictionaryReader
+  attr_reader :word_list
+
+  def self.next_word(word)
+    new.find_next_word(word)
+  end
+
```

Comment:
The class method ::next_word is the only method that needs to be called by the runner file.